sootup.java.core/src/main/java/sootup/java/core/JavaSootMethod.java

```diff
@@ -68,10 +68,7 @@ public boolean isStaticInitializer() {
 
   @Nonnull
   public Iterable<AnnotationUsage> getAnnotations(@Nonnull Optional<JavaView> view) {
-    annotations.forEach(e -> e.getAnnotation().getDefaultValues(view));
```

Comment:
@swissiety do you know if this line is important? Is it maybe needed because of the lazy loading?

Reply:
`getDefaultValues` is called in `resolveDefaultsForAnnotationType`, no need to call it twice.